media_driver/linux/common/os/mos_gpucontext_specific.cpp

```diff
@@ -909,3 +1049,44 @@ MOS_STATUS GpuContextSpecific::AllocateGPUStatusBuf()
     m_statusBufferResource = graphicsResource;
     return MOS_STATUS_SUCCESS;
 }
+
+#if 0
```

Comment:
I was confused about this code until I noticed the #if 0. :) You'll know if you need this in the pull request or not.

Reply:
removed.